examples/gnrc_lorawan/README.md

```diff
@@ -28,7 +28,11 @@ ifconfig 3 set appeui BBBBBBBBBBBBBBBB
 ifconfig 3 set appkey CCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC
 ifconfig 3 up
 ```
+If Chirpstack is being used, set AppEUI as follows
```

Comment:
I don't understand why using a Chirpstack server would need to clear the appeui.

Reply:
Actually it's not necessary to clear it. It just the fact that the default Application Server of Chirpstack ignores the AppEUI

Reply:
Yup as @jia200x  mentioned. There is no real purpose hence zeroed.

Reply:
maybe this should be reflected differently:
E.g `If Chirpstack is being used, the AppEUI is ignored`